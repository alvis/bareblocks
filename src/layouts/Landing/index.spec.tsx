/**
 *                            *** MIT LICENSE ***
 * -------------------------------------------------------------------------
 * This code may be modified and distributed under the MIT license.
 * See the LICENSE file for details.
 * -------------------------------------------------------------------------
 * @summary   A test for the landing page layout.
 *
 * @author    Alvis HT Tang <alvis@hilbert.space>
 * @license   MIT
 * @copyright Copyright (c) 2020 - All Rights Reserved.
 * -------------------------------------------------------------------------
 */

import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '.';

describe('Layout coomponent for landing', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Layout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
