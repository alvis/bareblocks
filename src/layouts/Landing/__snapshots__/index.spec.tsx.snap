// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout coomponent for landing should render correctly 1`] = `null`;
